opencog/opencog/spacetime/atom_types.script

```diff
@@ -15,6 +15,7 @@ IMAGINARY_STRUCTURE_NODE <- OBJECT_NODE
 TIME_NODE <- NODE
 TIME_DOMAIN_NODE <- NODE
 AT_TIME_LINK <- ORDERED_LINK
+AT_LOCATION_LINK <- ORDERED_LINK
```

Comment:
Im'm confused. This suggests that a large portion of opencog has been copied into this repo? This should be fixed ...